StyleCop.Analyzers/StyleCop.Analyzers/MaintainabilityRules/SA1119CodeFixProvider.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace StyleCop.Analyzers.MaintainabilityRules
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Composition;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeActions;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using Microsoft.CodeAnalysis.Formatting;
+
+
```

Comment:
These blank lines are getting inserted due to a bug in the Remove and Sort Usings feature in Visual Studio 2015 preview. After running this command, you should look at the result and see if it added the extra lines.